cmd/all-in-one/all_in_one_test.go

```diff
@@ -180,9 +179,11 @@ func getServicesAPIV3(t *testing.T) {
 	require.NoError(t, err)
 	body, _ := io.ReadAll(resp.Body)
 
-	var servicesResponse api_v3.GetServicesResponse
```

Comment:
This was unnecessary coupling. `api_v3` is not internal and not accessible from here.